plugins/activity-resources/src/components/Activity.svelte

```diff
@@ -24,6 +24,7 @@
 
   export let object: Doc
   export let fullSize: boolean = false
+  export let noCommentInput: boolean = false
```

Comment:
I would prefer not to have inverse value for properties.
```suggestion
  export let showCommenInput: boolean = true
```